packages/shared/src/lib/func.ts

```diff
@@ -1,2 +1,10 @@
 export const nextTick = (): Promise<unknown> =>
   new Promise((resolve) => setTimeout(resolve));
+
+export const safeJsonParse = <T = unknown>(data: string): T | string => {
```

Comment:
This name is not the most explicit name I think.
It's not about safe but about returning a default or a fallback value

Reply:
Makes total sense. I refactored it to `parseOrDefault`. Thank you! 🚢